src/pages/docs/chat/react-ui-components/components.mdx

```diff
@@ -0,0 +1,570 @@
+---
+title: "Ably Chat UI React Components"
+meta_description: "Comprehensive documentation for Ably Chat React UI Components."
+meta_keywords: "Ably, realtime, Chat UI, React, components, chat-message, chat-message-list, chat-window, message-input, sidebar, participant-list, avatar"
+---
+
+# Components
+
+This page documents the components available in the Ably Chat React UI Components library. These components provide a comprehensive UI for building chat applications with Ably.
+
+## Avatar <a id="avatar"/>
```

Comment:
I think we should order the components by their importance - e.g. `ChatWindow` should probably be top